src/components/sbb-tag-group/sbb-tag-group.e2e.ts

```diff
@@ -1,588 +1,558 @@
-        expect(changeSpy).not.toHaveReceivedEvent();
+        expect(tag1).to.have.attribute('checked');
+        expect(tag1.checked).to.be.equal(true);
+        expect(inputSpy.count).not.to.be.greaterThan(0);
+        expect(changeSpy.count).not.to.be.greaterThan(0);
       });
     });
 
     describe('one initialized checked tag', () => {
       beforeEach(async () => {
-        page = await newE2EPage();
-        await page.setContent(`
-      <sbb-tag-group multiple>
-        <sbb-tag id="sbb-tag-1" value="tag1">Tag 1</sbb-tag>
-        <sbb-tag id="sbb-tag-2" value="tag2" checked>Tag 2</sbb-tag>
-        <sbb-tag id="sbb-tag-3" value="tag3">Tag 3</sbb-tag>
-      </sbb-tag-group>
-    `);
-        element = await page.find('sbb-tag-group');
-        await page.waitForChanges();
+        element = await fixture(html`
+          <sbb-tag-group multiple>
+            <sbb-tag id="sbb-tag-1" value="tag1">Tag 1</sbb-tag>
+            <sbb-tag id="sbb-tag-2" value="tag2" checked>Tag 2</sbb-tag>
+            <sbb-tag id="sbb-tag-3" value="tag3">Tag 3</sbb-tag>
+          </sbb-tag-group>
+        `);
       });
 
       it('should have one activated tag', async () => {
-        expect((await element.findAll('sbb-tag[checked]')).length).toBe(1);
-        expect(await element.getProperty('value')).toEqual(['tag2']);
+        expect(element.querySelectorAll('sbb-tag[checked]').length).to.be.equal(1);
+        expect(element.value).to.be.eql(['tag2']);
       });
 
       it('should emit events and update value by unchecking manually', async () => {
-        const changeSpy = await page.spyOnEvent('change');
-        const inputSpy = await page.spyOnEvent('input');
-        const tag2 = await page.find('sbb-tag#sbb-tag-2');
+        const changeSpy = new EventSpy('change');
+        const inputSpy = new EventSpy('input');
+        const tag2 = document.querySelector('sbb-tag#sbb-tag-2') as SbbTag;
 
-        expect(tag2).toHaveAttribute('checked');
-        expect(await tag2.getProperty('checked')).toBe(true);
+        expect(tag2).to.have.attribute('checked');
+        expect(tag2.checked).to.be.equal(true);
 
-        await tag2.click();
-        await page.waitForChanges();
+        tag2.click();
+        await element.updateComplete;
 
-        expect(tag2).not.toHaveAttribute('checked');
-        expect(await tag2.getProperty('checked')).toBe(false);
+        expect(tag2).not.to.have.attribute('checked');
+        expect(tag2.checked).to.be.equal(false);
         await waitForCondition(() => inputSpy.events.length === 1);
-        expect(inputSpy).toHaveReceivedEventTimes(1);
+        expect(inputSpy.count).to.be.equal(1);
         await waitForCondition(() => changeSpy.events.length === 1);
-        expect(changeSpy).toHaveReceivedEventTimes(1);
-        expect(await element.getProperty('value')).toEqual([]);
+        expect(changeSpy.count).to.be.equal(1);
+        expect(element.value).to.be.empty;
```

Comment:
I think empty and empty array is not the same?

Reply:
It should work, but I can add an additional check (`.to.be.an('array').that.is.empty;`) as suggested by the [doc](https://www.chaijs.com/api/bdd/#method_empty:~:text=a%20target%E2%80%99s%20type.-,expect(%5B%5D).to.be.an(%27array%27).that.is.empty%3B,-Add%20.not).